### PR TITLE
db96 config staging and intermediate models

### PR DESCRIPTION
**Pull request by ian-r-rose**
This adds staging and intermediate models for the new config metadata loaded in #115 .

Some implementation notes:

1. Now that we have more than one source of VDS data (clearinghouse and db96), I've created a new "vds" schema for models that are agnostic as to which data source is being used. Eventually, we want to union the historical data with the ongoing data, and I view that as happening in the `vds` schema.
2. As in other places, the `int_vds__*_config` models implement an SCD-2 pattern with `_valid_from` and `_valid_to` for when each particular metadata row is active.
3. I've done some additional denormalization in `int_vds__detector_config` to also attach the station config to those rows. This is a pragmatic step because so many downstream metrics rely on having station-level metadata, even when working at the detector level. @JamesSLogan `int_vds__detector_config` is likely the model that you will want to join with the five-minute data for #214

### CODE REVIEW

File: transform/models/intermediate/vds/int_vds__controller_config.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    select * from {{ ref('stg_db96__controller_config_log') }}
+),
+
+config as (
+    select * from {{ ref('stg_db96__controller_config') }}
+),
+
+config_log_with_validity as (
+    select
+        *,
```
Comment (JamesSLogan) [inline transform/models/intermediate/vds/int_vds__controller_config.sql:13]: Does it make sense to exclude `time_id` here for the sake of eliminating redundant columns? (also in int_vds_station_config.sql, at least)

**Review by JamesSLogan — COMMENTED** [review]
The model documentation was very helpful 👌

File: transform/models/intermediate/vds/int_vds__controller_config.sql (hunk 1 above)
Comment (ian-r-rose) [reply transform/models/intermediate/vds/int_vds__controller_config.sql:13]: Yeah, I agree, I *almost* did that this morning, but was lazy

Comment (ian-r-rose) [reply transform/models/intermediate/vds/int_vds__controller_config.sql]: Done

**Review by JamesSLogan — APPROVED** [review]
Looks good to me

File: transform/models/_sources.yml [hunk 2]
```diff
@@ -155,3 +155,256 @@ sources:
             description: The name of the file in the clearinghouse where the metdata was obtained from.
           - name: CONTENT
             description: The XML content of the status metadata file.
+  - name: db96
+    database: "{{ env_var('DBT_RAW_DB', 'RAW_PRD') }}"
+    schema: db96
+    description: |
+      Vehicle Detector Station (VDS) (aka Station) related data obtained via data relay from
+      db96 within the Caltrans network
```
Comment (britt-allen) [inline transform/models/_sources.yml:163]: nit-pick - sentence missing a period.

Comment (britt-allen) [reply transform/models/_sources.yml:163]: I noticed a few other instances in this file too.

File: transform/models/staging/db96/stg_db96__controller_config.sql
Comment (britt-allen) [inline transform/models/staging/db96/stg_db96__controller_config.sql:1]: this file doesn't follow the dbt best practice, but if this is how all of our staging files are formatted i am fine with that.

**Review by britt-allen — APPROVED** [review]
Great work! Thank you for the detailed docs and carefully constructed models, Ian!

**Pull request merged**